spring-integration-amqp/src/main/java/org/springframework/integration/amqp/dsl/Amqp.java

```diff
@@ -25,6 +25,11 @@
 import org.springframework.integration.amqp.channel.PollableAmqpChannel;
 import org.springframework.integration.amqp.inbound.AmqpMessageSource.AmqpAckCallbackFactory;
 import org.springframework.lang.Nullable;
+import org.springframework.rabbit.stream.listener.StreamListenerContainer;
+import org.springframework.rabbit.stream.producer.RabbitStreamTemplate;
+
+import com.rabbitmq.stream.Codec;
+import com.rabbitmq.stream.Environment;
```

Comment:
I saw these all as an `optional` API in the dependencies, so this `Amqp` factory class is going to fail when `srteam` is not available. No? What do I miss?